hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/utils/HddsVolumeUtil.java

```diff
@@ -212,6 +212,9 @@ public static boolean checkVolume(HddsVolume hddsVolume, String scmId, String
     } else {
       // The hdds root dir should always have 2 files. One is Version file
       // and other is SCM directory.
+      logger.error("The hdds root dir {} should always have 2 files. " +
+              "One is Version file and other is SCM directory.",
```

Comment:
Let's add a more actionable message: "Please remove any other extra files from the directory so that DataNode startup can proceed."

Reply:
Thanks @jojochuang  for reviewing and committing it.